src/heroes/doomfist/slam.opy

```diff
@@ -209,9 +215,10 @@ rule "[doomfist/slam.opy]: Calculate slam damage based on air time":
     eventPlayer.slam_damage = 0
     showSlamDamageGui()
     waitUntil(not eventPlayer.isOnGround(), Math.INFINITY)
-    if eventPlayer.slam_to_use == SlamType.GROUND: # Makes ground slam damage 49    
-        wait()
+    #if eventPlayer.slam_to_use == SlamType.GROUND: # Makes ground slam damage 49    
+        #wait()
     chase(eventPlayer.slam_damage, OW1_DOOMFIST_SLAM_DAMAGE_MAX, rate=OW1_DOOMFIST_SEISMIC_SLAM_DAMAGE_RATE, ChaseReeval.NONE)
+    wait(0.25)
```

Comment:
can you explain the changes here?

Reply:
i removed the action to add wait a tick on ground slam as the code wont have accurate damage values if i kept it. and  just like the other "wait 0.25", the wait is to wait the minimum slam time before being able to stop the damage from increasing, else it will be able to stop slam damage when youre actually still slamming